pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -452,4 +522,60 @@ static void _session_to_ep(const session_t *session, sock_udp_ep_t *ep)
     memcpy(&ep->addr.ipv6, &session->addr, sizeof(ipv6_addr_t));
 }
 
+#ifdef SOCK_HAS_ASYNC
+void _udp_cb(sock_udp_t *udp_sock, sock_async_flags_t flags, void *ctx)
+{
+    if (flags & SOCK_ASYNC_MSG_RECV) {
+        sock_dtls_t *sock = ctx;
+        sock_dtls_session_t remote;
+        void *data;
+        void *data_ctx;
```

Comment:
This fix the FAILED_ASSERTION in https://github.com/RIOT-OS/RIOT/pull/12907#issuecomment-616111122.

```suggestion
        void *data = NULL;
        void *data_ctx = NULL;
```